### PR TITLE
Vote 349/reg dates bn

### PR DESCRIPTION
**Pull request by SamiraMSadat**
Updated the Bengali pages to include the 2022 registration dates for each state.
1. General Election date
2. Online Registration
3.  Postmarked or Recieved mail-in reg
4. In Person Registration

https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-349

### CODE REVIEW

File: data/translations/es/state-data.json [hunk 1]
```diff
@@ -225,6 +225,7 @@
     "online_deadline": "hasta 20 días antes del día de las elecciones",
     "mail_received_deadline": "",
     "mail_postmarked_deadline": "debe tener el sello del correo 30 días antes del día de las elecciones",
+    "dates_electionday": "শনিবার, ১০ ডিসেম্বর, ২০২২",
```
Comment (rayestrada) [inline data/translations/es/state-data.json:228]: This looks like it was an error. Spanish data files should not be modified as part of this branch.

File: content/es/registrar/la.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ ip_deadline = "hasta 30 días antes del día de las elecciones"
 online_deadline = "hasta 20 días antes del día de las elecciones"
 mail_received_deadline = ""
 mail_postmarked_deadline = "debe tener el sello del correo 30 días antes del día de las elecciones"
+dates_electionday = "শনিবার, ১০ ডিসেম্বর, ২০২২"
```
Comment (rayestrada) [inline content/es/registrar/la.md:15]: This looks like it was an error. Spanish data files should not be modified as part of this branch.

File: data/translations/bn/state-data.json [hunk 3]
```diff
@@ -3,311 +3,407 @@
...
+        "confirm_registration_link": "https://ova.elections.il.gov/RegistrationLookup.aspx",
+        "default_ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
+        "default_online_deadline": "নির্বাচনের ১৬ দিন আগে",
+        "default_mail_postmarked_deadline": "নির্বাচনের দিন ২৮ দিন আগে পোস্টমার্ক করা আবশ্যক"
     },
     "in": {
         "hp_link": "https://www.in.gov/sos/elections/",
         "registration_link": "https://indianavoters.in.gov/",
         "more_info_link": "https://www.in.gov/sos/elections/2403.htm",
-        "ip_deadline": "নির্বাচনের দিন ২৯ দিন আগে",
```
Comment (rayestrada) [inline data/translations/bn/state-data.json:145]: We need to keep the current `ip_deadline` as a `default_` value for fallback purposes.

File: data/translations/bn/state-data.json [hunk 4]
```diff
@@ -3,311 +3,407 @@
...
+        "mail_postmarked_deadline": "পোস্ট চিহ্ন করা আবশ্যক (মঙ্গলবার, ১১ অক্টোবর, ২০২২)",
+        "confirm_registration_link": "https://sosmt.gov/elections/vote/",
+        "default_ip_deadline": "নির্বাচনের দিন পর্যন্ত এবং সহ উপলব্ধ",
+        "default_mail_postmarked_deadline": "নির্বাচনী দিনের ৩০ দিন আগে পোস্টমার্ক করা আবশ্যক"
     },
     "ne": {
         "hp_link": "https://sos.nebraska.gov/elections/2020-elections",
         "registration_link": "https://www.nebraska.gov/apps-sos-voter-registration/",
         "more_info_link": "https://sos.nebraska.gov/elections/voter-forms",
-        "ip_deadline": "নির্বাচনের আগের দ্বিতীয় শুক্রবার",
```
Comment (rayestrada) [inline data/translations/bn/state-data.json:256]: We need to keep the current `ip_deadline` as a `default_` value for fallback purposes.

File: data/translations/bn/state-data.json [hunk 5]
```diff
@@ -3,311 +3,407 @@
...
+        "mail_postmarked_deadline": "পোস্ট চিহ্ন করা আবশ্যক (শুক্রবার, ২১ অক্টোবর, ২০২২)",
+        "confirm_registration_link": "https://www.votercheck.necvr.ne.gov/voterview",
+        "default_online_deadline": "নির্বাচনের আগের তৃতীয় শুক্রবার",
+        "default_mail_postmarked_deadline": "নির্বাচনের দিন আগে তৃতীয় শুক্রবারের মধ্যে পোস্টমার্ক করতে হবে"
     },
     "nv": {
         "hp_link": "https://www.nvsos.gov/sos/elections",
         "registration_link": "https://www.nvsos.gov/SOSVoterServices/start.aspx",
         "more_info_link": "https://www.nvsos.gov/sos/elections/voters/registering-to-vote",
-        "ip_deadline": "নির্বাচনের আগে চতুর্থ মঙ্গলবার",
```
Comment (rayestrada) [inline data/translations/bn/state-data.json:265]: We need to keep the current `ip_deadline` as a `default_` value for fallback purposes.

**Review by rayestrada — CHANGES_REQUESTED** [review]
@SamiraMSadat We need to complete a few minor revisions. Thanks.

File: data/translations/es/state-data.json (hunk 1 above)
Comment (SamiraMSadat) [reply data/translations/es/state-data.json:228]: Great catch! This should be in the Bengali file. Fixed :)

File: content/es/registrar/la.md (hunk 2 above)
Comment (SamiraMSadat) [reply content/es/registrar/la.md:15]: Fixed

File: data/translations/bn/state-data.json (hunk 3 above)
Comment (SamiraMSadat) [reply data/translations/bn/state-data.json:145]: fixed

File: data/translations/bn/state-data.json (hunk 4 above)
Comment (SamiraMSadat) [reply data/translations/bn/state-data.json:256]: Fixed

File: data/translations/bn/state-data.json (hunk 5 above)
Comment (SamiraMSadat) [reply data/translations/bn/state-data.json:265]: fixed

**Review by rayestrada — DISMISSED** [review]
Looks good thanks!

**Review by rayestrada — APPROVED** [review]

**Pull request merged**